src/components/CategoriesExpandableList.tsx

```diff
@@ -1,35 +1,51 @@
-import { AccordionPanel, AccordionTitle, AccordionContent, Accordion } from "flowbite-react";
+import {
+  AccordionPanel,
+  AccordionTitle,
+  AccordionContent,
+  Accordion,
+} from "flowbite-react";
 import { categories } from "./categories";
 
-export const CategoriesExpandableList = () => {
-
+const CategoriesExpandableList = () => {
   const renderSubcategories = (categ: any) => {
```

Comment:
- Prefer early return statements
```
if(x) return X 
if(y) return Y 
return Z
```

 instead of ternary operators, for any content that does not fit in a very short line of code(in most cases an assignment)

- avoid nesting ternaries in a layout

-avoid returning empty fragments